content/lessons/_index.md

```diff
@@ -0,0 +1,3 @@
+### Lessons
+
+{{< lesson-gallery show="topics" >}}
```

Comment:
This is related to the comment below:

https://github.com/3b1b/3Blue1Brown.com/pull/42#discussion_r638314662

Reply:
This won't work as the lesson gallery should be passed to the `main` partial for processing and be under the `<main>` element. Also I think it belongs as a shortcode because it's a component that you insert into content. You may want to add explanatory text or images above or below the component (that belong in the markdown). I don't think you can split the main content of the markdown into separate pieces and put them in different places in the template?

Reply:
Oh, that makes sense. Like you might want to insert the lesson gallery into the homepage as a shortcode.

>  I don't think you can split the main content of the markdown into separate pieces and put them in different places in the template?

Yeah the main content of the markdown file is kept together. One thing worth mentioning here is that you can structure a page into multiple markdown files using [Page Resources](https://gohugo.io/content-management/page-resources/) when the page is a bundle.